pkg/controller/seed-controller-manager/initial-application-installation-controller/controller.go

```diff
@@ -190,10 +190,35 @@ func (r *Reconciler) reconcile(ctx context.Context, cluster *kubermaticv1.Cluste
 }
 
 func (r *Reconciler) createInitialApplicationInstallations(ctx context.Context, client ctrlruntimeclient.Client, application apiv1.Application, cluster *kubermaticv1.Cluster) error {
```

Comment:
Unrelated, but can we rename this to `createInitialApplicationInstallation` (singular)?

Reply:
Sure, updated